manifests/redis-deployment-dev.yaml

```diff
@@ -1,51 +0,0 @@
-apiVersion: v1
```

Comment:
You seem to have deleted a lot of files in `/manifests`. Are they no longer needed?

Reply:
yeah - so these were the manifests that we used to deploy application components before helm, and they had just been hanging around them. Instead of deleting the whole folder I'm using it to store things that we use during cluster setup - like our ingress definition, our cert-manager config, and a configuration step that we use to make installing redis from helm easier.

Reply:
Awesome. Love it!